### ISSUE TITLE
Output is written to the input file, with -o ignored

### ISSUE THREAD

**Issue by Thunderforge** [tjhancocks/kestrel-development-kit#26]
**Describe the bug/issue**

When running `kas`, the output location (specified by `-o`) is ignored and output instead goes to the input file (specified by `-f`).

**To Reproduce**

1. In the command line at the project root, run `make` to generate the `build` directory
1. Run `cd build`
1. Run `./kas -o plugin.kdat -f "../examples/kdl/test.kdl"` ([per the Readme](https://github.com/tjhancocks/kestrel-development-kit/commit/7434b2dba83a81886ed80e6c394840d0bf849e51#diff-04c6e90faac2675aa89e2176d2eec7d8R21))

Note that the output is as follows:

> Hello, World
> Saving data to ../examples/kdl/test.kdl

As you can see, the output was sent to the input file, overwriting it.

**Expected behavior**
In the above example, I expect to see:

> Hello, World
> Saving data to plugin.kdat

In other words, the output should go to what was passed in with `-o`.

**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
